api/src/main/java/io/strimzi/api/kafka/model/KafkaAuthorization.java

```diff
@@ -41,6 +41,13 @@
             "`custom` authorization type uses user-provided implementation for authorization.")
     public abstract String getType();
 
+    /**
+     * Indicates whether the Authorizer supports the Kafka Admin API to manage ACLs or not.
+     *
+     * @return True if ACLs can be managed using Kafka Admin API. False otherwise.
+     */
+    public abstract boolean supportsAdminApi();
```

Comment:
Putting this in the baseclass means that the schema will suggest that each of the subtypes support this property, which is not true: Only the custom one supports a property which can actually be written. The schema ought to reflect this fact.

Reply:
That will happen anyway, or? The Schema does not distinguish to which type the field belongs. You can see all the fields in one long list and I do not think it is there because of this but because of the field in `KafkaAuthorizationCustom`.

Reply:
Oh yeah, you're right, because of the rubbish way we try to simulate inheritance in the JSON schema. Yuck.

Reply:
So, I guess there is not much we can do about it here and it should not matter?